packages/core/src/storage/vectorStore/AstraDBVectorStore.ts

```diff
@@ -102,12 +105,13 @@ export class AstraDBVectorStore implements VectorStore {
     if (!nodes || nodes.length === 0) {
       return [];
     }
+
     const dataToInsert = nodes.map((node) => {
       return {
-        _id: node.id_,
-        $vector: node.getEmbedding(),
-        content: node.getContent(MetadataMode.ALL),
-        metadata: node.metadata,
+        $vector: node.embedding,
```

Comment:
seems to me like this new format is not compatible with Python, see:

https://github.com/run-llama/llama_index/blob/7cc6b1db27392ff07ecd5d95c5e00a743e212e93/llama_index/vector_stores/astra.py#L115-L129

suggestions:
1. use `nodeToMetadata`
2. defaults for keys being `_id`, `content` and `metadata`

Reply:
@marcusschiesser thanks for the feedback. I made some changes to better handle metadata and align w/ the python version. I also was hoping to handle for the existing vector db table that doesn't necessarily have a metadata column.

For example:
![image](https://github.com/run-llama/LlamaIndexTS/assets/6100513/3b1614e6-e010-437c-9a6a-d84f56e59acc)


Please let me know if my handling in `query` aligns with how llamaindex expects to handle this situation